docs/reuse/apps/opentelemetry/logs-advance-option-otel.md

```diff
@@ -0,0 +1,6 @@
+**Advance options** for log collection can be used as follows : 
+
+    **Timestamp Format**. By default, Sumo Logic will automatically detect the timestamp format of your logs. However, you can manually specify a timestamp format for a source by specifying the following :
+        - **Timestamp locator** : This will be a [Go regular expression](https://github.com/google/re2/wiki/Syntax) which should have timestamp matched using a named capture group. Name of this captured group should be “timestamp_field”
```

Comment:
```suggestion
        - **Timestamp locator**. This will be a [Go regular expression](https://github.com/google/re2/wiki/Syntax) which should have timestamp matched using a named capture group. Name of this captured group should be “timestamp_field”.
```